lib/pbench/test/unit/server/test_unpack_tarballs.py

```diff
@@ -18,10 +19,39 @@ def make_logger(server_config):
     return get_pbench_logger("TEST", server_config)
 
 
+class MockConfig:
+    ARCHIVE = Path("/mock/ARCHIVE")
+    INCOMING = Path("/mock/INCOMING")
+    RESULTS = Path("/mock/RESULTS")
+    TS = "my_timestamp"
+
+    def get(section: str, option: str, fallback=None) -> str:
+        """returns value of the given `section` and `option`"""
+
+        _sections = {
+            "pbench-server": {
+                "unpacked-states": "TO-INDEX",
+            },
+        }
+
+        if section not in _sections:
+            raise NoSectionError(section)
+        if option not in _sections[section]:
+            raise NoOptionError(option, section)
+        value = _sections[section][option]
+        if not value:
+            if not fallback:
+                raise ValueError(option, section)
+            else:
+                return fallback
+        return _sections[section][option]
```

Comment:
Given that this is a mock and we know what it should be called for, I would go with a much more targeted approach...something like:
```
    def get(section: str, option: str, fallback=None) -> str:
        """Mock function for Config.get()"""

        assert section == "pbench-server"
        assert option == "unpacked-states"
        return "TO-INDEX"
```